.eslintignore

```diff
@@ -10,7 +10,7 @@ bower_components
 /html_docs
 /src/plugins/data/common/es_query/kuery/ast/_generated_/**
 /src/plugins/vis_type_timelion/public/_generated_/**
-src/legacy/core_plugins/vis_type_vislib/public/vislib/__tests__/lib/fixtures/mock_data
+/src/plugins/vis_type_vislib/public/fixtures/mock_data
```

Comment:
Might be a good opportunity to fix the formatting in these files

Reply:
Okay, I've done it in a separate [commit](https://github.com/elastic/kibana/pull/63963/commits/05a5e9e6fb2c0183c3c3a024cb32714f15f1168b) in case of any concerns (the diff has increased significantly)